.svnignore

```diff
@@ -42,3 +42,4 @@ docker
 bin/pre-commit-hook.js
 bin/travis_install.sh
 yarn-error.log
+extensions/**/*.{js,jsx,gif,png,css,jpg,jpeg,sass,scss}
```

Comment:
```suggestion
extensions/**/*.{js,jsx,json,gif,png,css,jpg,jpeg,sass,scss}
```

Reply:
What do you think about doing a reverse?

At least this works in bash:
```bash
ls !(*.php)
```

On second thought, that might be potentially harmful vs explicitly defining what we don't want?

Reply:
I'd rather be explicit in svnignore as it can be really surprising. Something works great in GItHub and unless someone tests that explicit feature using a beta that was shipped in SVN we could have a pretty big gap in the production release.

To help future groking, I'd love for svnignore comments (I believe `#` preceding the line should work here too) to explain why these files are ignored. Someone new to JP development may be confused why we're ignoring jpg, css, etc.